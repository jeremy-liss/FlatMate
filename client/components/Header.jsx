import React from 'react'

const Header = () => (
  <div className='header'>
    <h1>Flat Mate </h1>
  </div>
)

export default Header
