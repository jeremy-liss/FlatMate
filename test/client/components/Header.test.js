import React from 'react'
import render from 'react-test-renderer'

import Header from '../../../client/components/Header.jsx'

it('Does the header read Flat Mate?', ()=> {
  const tree = render.create(
    <link page="../../../client/components/Header"></link>
  ).toJSON()
  expect(tree).toMatchSnapshot()
  })
