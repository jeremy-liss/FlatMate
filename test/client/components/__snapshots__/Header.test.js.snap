// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Does the header read Flat Mate? 1`] = `
<link
  page="../../../client/components/Header"
/>
`;
